taf/tools/keystore/__init__.py

```diff
@@ -0,0 +1,36 @@
+import click
+import taf.developer_tool as developer_tool
+
+
+def attach_to_group(group):
+    @group.group()
+    def keystore():
+        pass
+
+    @keystore.command()
+    @click.argument("path")
+    @click.argument("keys-description")
+    def generate(path, keys_description):
+        """Generate keystore files and save them to the specified location given a dictionary containing
+        information about each role - total number of keys, their lengths and keystore files' passwords.
+        It is necessary to either directly specify this dictionary when calling this command or
+        to provide a path to a `.json` file which contains the needed information.
+
```

Comment:
You could use `\b` to prevent click text rewrapping.

https://click.palletsprojects.com/en/7.x/documentation/#preventing-rewrapping